hack/jenkins/minikube_deb_install_test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script builds the minikube binary for all 3 platforms and uploads them.
+# This is to done as part of the CI tests for Github PRs
+
+# The script expects the following env variables:
+# bucket: The GCP bucket to get .deb files from
+# debs: .deb packages to test
+
+set -eu -o pipefail
+
+trap exit SIGINT
+
+for pkg in ${debs[@]}; do
+    gsutil cp -r "gs://${bucket}/${pkg}.deb" .
+done  
+
+declare -ra distros=(\
```

Comment:
is this file used anywhere ?

minikube_deb_install_test.sh

Reply:
Initially was used in Jenkins, now we switched to GitHub actions, so it's safe to delete this file. done.